app/src/main/res/values-ro/strings.xml

```diff
@@ -85,7 +90,7 @@
     <string name="sbc_before_color">Text înainte de ceas</string>
     <string name="sbc_color">Ceas</string>
     <string name="sbc_after_color">Text după ceas</string>
-    <string name="network_settings_pref">Setări monitorizare rețea…</string>
+    <string name="network_settings_pref">Setări monitorizare rețea...</string>
```

Comment:
Please revert the "…" to "..." change. Android Studio defines "…" as correct implementation and always asks us to change all "..." to "…"

Reply:
Done